### PR TITLE
Vagrant refactoring

### PR DESCRIPTION
**Pull request by yuramelesh**
Added:
- pr checking for "New" status
- vagrant ENV var import
- bundle install

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+  source ~/.profile && if [ -z ${SEE_THROUGH_HOME_PATH+x} ]; then
+        echo "export SEE_THROUGH_HOME_PATH=#{SEE_THROUGH_HOME_PATH}" >> .profile
+  fi
+
+  source ~/.profile && if [ -z ${SEE_THROUGH_TOKEN+x} ]; then
+        echo "export SEE_THROUGH_TOKEN=#{SEE_THROUGH_TOKEN}" >> .profile
+  fi
+
+  source ~/.profile && if [ -z ${SEE_THROUGH_EMAIL+x} ]; then
```
Comment (osklyarenko) [inline Vagrantfile]: what does `+x` mean?

File: copy_project_to_server.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```
Comment (osklyarenko) [inline copy_project_to_server.sh]: why do I see this file again. it's gone.

File: daily_report_mail.rb [hunk 3]
```diff
@@ -98,7 +93,10 @@ def get_new_pr other_block
     new_pr << '<hr><h2>New pull requests</h2>'
     other_block.each do |i|
       if i[:index] == 3
```
Comment (osklyarenko) [inline daily_report_mail.rb]: what's that? why do we need this? it's unreadable.

File: daily_report_mail.rb [hunk 4]
```diff
@@ -161,7 +159,9 @@ def create_mail_message user_to, repo
 
   #{your_problem_pr}
   #{recently_merged}
-  #{new_pr}
+  #{if new_pr.length > 36
```
Comment (osklyarenko) [inline daily_report_mail.rb]: `36` what does this magic number mean?

File: ops/provision.sh [hunk 5]
```diff
@@ -1,35 +1,32 @@
...
-echo "------------------------------------------------------------------"
-echo "--------------------- Add cron jobs ------------------------------" 
-echo "------------------------------------------------------------------"
-
-echo "0 * * * * /home/ubuntu/daily_report.sh
-*/15 * * * * /home/ubuntu/conflict_checker.sh" | crontab -
+
+echo "------ Install gems ------"
+
+sudo bundle install
```
Comment (osklyarenko) [inline ops/provision.sh]: did you check it is actually working?
A Gemfile should be immediately available to Bundler to pick it up.
http://bundler.io/v1.11/bundle_install.html

File: time_class.rb [hunk 6]
```diff
@@ -39,7 +39,7 @@ def check_time (user_tz)
     return false
   end
 
-  def check_sent_at (sent_at)
+  def check_24_hours_past (sent_at)
```
Comment (osklyarenko) [inline time_class.rb]: pass hrs in a parameter. don't hardcode it.

File: daily_report.rb [hunk 7]
```diff
@@ -37,26 +36,22 @@ def mail_sending (repo, user)
...
+      if user.enable
+        repo = repos.repository_name
+        @controller.get_pr repo
 
         if @time.check_time(user.notify_at.to_s)
           repo = repos.repository_name
-          if true #!data_existing
-            get_pr repo
+
+          if !data_existing
```
Comment (osklyarenko) [inline daily_report.rb]: unless

File: Vagrantfile (hunk 1 above)
Comment (zdmytriv) [inline Vagrantfile]: O_o

**Pull request closed without merging**